app/assets/stylesheets/components/_troubleshooting-options.scss

```diff
@@ -5,7 +5,7 @@
     margin-bottom: 0;
   }
 
-  &::before {
+  &:not(.troubleshooting-options__no-bar)::before {
```

Comment:
I feel like a negative class like this is kinda weird... but I like that we have default behavior + add a class to change it? open to flip this around if needed

Reply:
Also, in the BEM naming scheme, "no bar" sounds to me like a modifier state, so I would expect the class to be `troubleshooting-options--no-bar`.

Reply:
- BEM name in d2670cfbe86a7ab004325ca04f59d9b0b49557c5
- simplified SCSS in 035fa43e3d893f6285d16974230f3ae54099afd2